downstream/modules/platform/ref-controller-variables.adoc

```diff
@@ -0,0 +1,113 @@
+Set to `False` to disable.
+
+Default = `‘true’`
+| *`node_state`* | _Optional_
+
+The status of a node or group of nodes. 
+Valid options are `‘active’`, `'deprovision'` to remove a node from a cluster or `'iso_migrate'` to migrate a legacy isolated node to an execution node.
+
+The default is `‘active’`.
+| *`node_type`* | For `[automationcontroller]` group.
+
+Two valid `node_types` can be assigned for this group.
+
+A `node_type='control'` implies that the node will only be able to run project and inventory updates, but not regular jobs.
+
+A `node_type='hybrid'` has the ability to run everything.
+
+The default for this group is `'hybrid'`.
+
+For `[execution_nodes]` group
+
+Two valid node_types can be assigned for this group.
+
+A `node_type='hop'` implies that the node forwards jobs to an execution node.
+
+A `node_type='execution'` implies that the node can run jobs.
+
+The default for this group is `'execution'`.
+a| *`peers`* | Peer relationships define node-to-node connections. 
+
+For example 
+
+`[automationcontroller:vars]
+node_type=control
+peers=instance_group_local`
+| *`pg_database`* | The name of the postgres database.
+
+The default is `‘awx’`.
+| *`pg_host`* | The postgreSQL host, which can be an externally managed database.
+| *`pg_password`* | The password for the postgreSQL database.
+
+Do not use special characters for `pg_password`. 
+They can cause the password to fail.
+
+NOTE
+
+You no longer need to provide a `pg_hashed_password` in your inventory file at the time of installation because postgreSQL 13 can now store user passwords more securely. 
+
+When you supply `pg_password`` in the inventory file for the installer, postgreSQL uses the SCRAM-SHA-256 hash to secure that password as part of the installation process.
+| *`pg_port`* | The postgreSQL port to use.
+
+The default is 5432
+| *`pg_ssl_mode`* | One of `'prefer'`, `'verify-full'`, ? ?
+
+Set to `'verify-full'` for client-side enforced SSL.
+
+The default is `‘prefer’`.
+| *`pg_username`* | Your postgres database username.
+
+The default is `‘awx’`.
+| *`postgres_use_cert`* | Location of postgres user certificate. 
+
+`'/path/to/pgsql.crt'`
+| *`postgres_use_key`* | Location of postgres user key. 
+
+`'/path/to/pgsql.key'`
+| *`postgres_use_ssl`* | TBD
+| *`receptor_listener_port`* | TBD
+| *`sso_console_admin_password`* | TBD
+| *`sso_console_keystore_file`* | Keystore file to install in SSO node.
+
+`'/path/to/sso.jks'`
+| *`sso_host`* | {CatalogNameStart} requires SSO and SSO admin credentials for
+authentication. 
+SSO administration credentials are also required to set Catalog specific
+roles needed for the application. 
+
+If SSO is not provided in the inventory for
+configuration, then the SSO host needs to be defined with this variable.
+| *`sso_keystore_password`* | A keystore password is required for https enabled SSO.
+
+The default install deploys SSO with `sso_use_https=True`
+| *`sso_use_https`* | TBD
```

Comment:
Those sso_xxx are not for Controller, but for Hub and/or Catalog.  Maybe a different section should be created for SSO and Hub/Catalog section reference that (so we don't need to duplicate in Hub and Catalog)??

I was working with @ariordan-redhat on SSO related vars today... hopefully you two can discuss and reorganize this area.